react/__mocks__/vtex.store-components/BuyButton.js

```diff
@@ -3,10 +3,8 @@ import React, { Component } from 'react'
 /**
  * BuyButton Mocked Component.
  */
-export default class BuyButton extends Component {
-  render() {
-    return (
-      <div>BuyButton</div>
-    )
-  }
-}
\ No newline at end of file
+const BuyButton = ({ children }) => {
+  return <div className="buy-button-mock">{children}</div>
```

Comment:
Add handling `className` prop to this mock.
```suggestion
  return <div className={`buy-button-mock ${className}`}>{children}</div>
```